src/helpers.ts

```diff
@@ -13,22 +13,22 @@ export function getkubectlDownloadURL(version: string, arch: string): string {
    switch (os.type()) {
       case 'Linux':
          return util.format(
-            'https://storage.googleapis.com/kubernetes-release/release/%s/bin/linux/%s/kubectl',
+            'https://dl.k8s.io/release/v%s/bin/linux/%s/kubectl',            
```

Comment:
potentially nit: is there a reason why we're going with util.format as opposed to template strings?

Reply:
Honestly, I just updated the string in the existing code.  I'm happy to shift it to template strings if you want.  It would probably be easier to read.

Reply:
let's move to template strings😃readability is always a plus

Reply:
updated.  Thank you!